jscomp/others/belt_MutableSetInt.mli

```diff
@@ -30,95 +30,605 @@
   and identity is not needed(using the built-in one)
 
   **See** [`Belt.MutableSet`]()
+
+  This module is [Belt.MutableSet](mutable-set) specialized with key type to be a `int` type.
+  It is more efficient in general, the API is the same with [Belt.MutableSet](mutable-set) except its key type is fixed, and identity is not needed (using the built-in one).
 *)
 
 
 # 38 "others/setm.cppo.mli"
 type value = int
-  
+(**
+  ```res prelude
+  type value = int
+  ```
+*)
+
 # 42 "others/setm.cppo.mli"
   (** The type of the set elements. *)
 
 
 type t
-(** The type of sets. *)
+(**
+  ```res prelude
+  type t
+  ```
+  The type of sets.
+*)
 
 val make: unit -> t
+(**
```

Comment:
The `res sig` codefences used to be our replacements for actual `val make: unit -> t` signatures.

Omit all the `res sig` examples if they resemble the function signature.